src/dso_api/dynamic_api/middleware.py

```diff
@@ -55,16 +52,3 @@ def process_view(self, request, view_func, view_args, view_kwargs):
                     )
 
         return None
-
-
-class AuthProfileMiddleware(BaseMiddleware):
```

Comment:
Moet deze echt weg?

Reply:
Het probleem is dat de auth_profile wordt geïnitialiseerd met een WSGI Request. Een Rest framework request geeft meer informatie. Info die ik nodig heb. Daarom heb ik AuthProfile(request) verplaatst de initial() van de view. Dat maakt dat deze nu overbodig is.